services/app-api/src/postgres/questionResponse/findAllQuestionsByHealthPlanPackage.ts

```diff
@@ -39,10 +43,7 @@ export async function findAllQuestionsByHealthPlanPackage(
                 ...question.addedBy,
                 stateAssignments: [],
             } as CMSUserType,
-            responses: question.responses.map((response) => ({
-                ...response,
-                addedBy: response.addedBy as StateUserType,
-            })),
+            responses: question.responses as QuestionResponseType[],
```

Comment:
My editor is showing a type error for 'questions' on line 40.
  Property 'division' is missing in type '{ addedBy: CMSUserType; responses: QuestionResponseType[]; id: string; createdAt: Date; updatedAt: Date; pkgID: string; addedByUserID: string; documents: QuestionDocument[]; }' but required in type 'Question'.ts(2322)

Reply:
Sometimes this happens to be me too, I have to deploy the app locally and then eventually it will catch up.